platforms/tktrex/backend/bin/parser.ts

```diff
@@ -77,6 +78,10 @@ const run = async (): Promise<void> => {
       write: mongoW,
     };
 
+    const errorReporter = FixtureReporter(
+      path.resolve(process.cwd(), `parsers/__tests__/fixtures`)
```

Comment:
Can we make the `"parsers/__tests__/fixtures"` string a constant variable or something?

Reply:
better an `env` variable?

Reply:
Done